.dev/up_deps.jl

```diff
@@ -0,0 +1,27 @@
+#=
+A simple script for updating the manifest
+files in all of our environments.
+=#
+
+root = dirname(@__DIR__)
```

Comment:
this is not the correct root?

Reply:
ah nvm I see that you are taking the dirname of the dir which is a bit confusing